### PR TITLE
1st Migration from altv.stuyk.com

### PR DESCRIPTION
**Pull request by Stuyk**
Migrating documentation from altv.stuyk.com to the official docs repository.

Includes the following:
* snippets
* alt-server
* alt-client
* Player Properties / Functions for Client
* Player Properties / Functions for Server
* Data Tables
* Bootcamp Tutorials
* Table of Contents Update

### CODE REVIEW

File: docs/articles/concepts/server-events.md [hunk 1]
```diff
@@ -0,0 +1,714 @@
+# Server Events
```
Comment (Lhoerion) [inline docs/articles/concepts/server-events.md:1]: This articles reinvents the wheel as well

File: docs/articles/concepts/ragemp-migration.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Migrating from RAGE:MP
```
Comment (Lhoerion) [inline docs/articles/concepts/ragemp-migration.md:1]: I'm not fond of articles of such type, it becomes outdated and in result obsolete very quickly

Comment (Lhoerion) [inline docs/articles/concepts/ragemp-migration.md:1]: Obsolete article

File: docs/articles/concepts/server-events.md (hunk 1 above)
Comment (LeonMrBonnie) [reply docs/articles/concepts/server-events.md:1]: These should rather be put in the typings directly with `@example` tag

File: docs/articles/tables/cause-of-death.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
+
```
Comment (Lhoerion) [inline docs/articles/tables/cause-of-death.md:1]: Definitely not

File: docs/articles/tables/explosions.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+
```
Comment (Lhoerion) [inline docs/articles/tables/explosions.md:1]: Definitely not

File: docs/articles/tables/controls.md [hunk 5]
```diff
@@ -0,0 +1,425 @@
+# Input Groups & Controls
```
Comment (Lhoerion) [inline docs/articles/tables/controls.md:1]: Definitely not

File: docs/articles/tables/index.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# Tables
```
Comment (Lhoerion) [inline docs/articles/tables/index.md:1]: Definitely not

File: docs/articles/tables/frontend-sounds.md [hunk 7]
```diff
@@ -0,0 +1,488 @@
+# Frontend Sounds
```
Comment (Lhoerion) [inline docs/articles/tables/frontend-sounds.md:1]: Definitely not

File: docs/articles/tables/ipls.md [hunk 8]
```diff
@@ -0,0 +1,172 @@
+# Missing Interiors IPLs
```
Comment (Lhoerion) [inline docs/articles/tables/ipls.md:1]: Definitely not

File: docs/articles/tables/ped-bones.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
+
```
Comment (Lhoerion) [inline docs/articles/tables/ped-bones.md:1]: Definitely not

File: docs/articles/tables/vehicle-mods.md [hunk 10]
```diff
@@ -0,0 +1,234 @@
+# Vehicle Mods
```
Comment (Lhoerion) [inline docs/articles/tables/vehicle-mods.md:1]: Definitely not

File: docs/articles/tables/vehicle-seats.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
+# alt:V Vehicle Seats
```
Comment (Lhoerion) [inline docs/articles/tables/vehicle-seats.md:1]: Definitely not

File: docs/articles/tables/vehicle-window-tint.md [hunk 12]
```diff
@@ -0,0 +1,13 @@
+# Vehicle Window Tints
```
Comment (Lhoerion) [inline docs/articles/tables/vehicle-window-tint.md:1]: Definitely not

File: docs/articles/tables/weapon-components.md [hunk 13]
```diff
@@ -0,0 +1,816 @@
+# Weapon Components
```
Comment (Lhoerion) [inline docs/articles/tables/weapon-components.md:1]: Definitely not

File: docs/articles/tables/weapon-pickups.md [hunk 14]
```diff
@@ -0,0 +1,149 @@
+# Weapon Pickups
```
Comment (Lhoerion) [inline docs/articles/tables/weapon-pickups.md:1]: Definitely not

File: docs/articles/tables/weapon-tint.md [hunk 15]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (Lhoerion) [inline docs/articles/tables/weapon-tint.md:1]: Definitely not

File: docs/articles/tables/weather.md [hunk 16]
```diff
@@ -0,0 +1,21 @@
+# Weather IDs
```
Comment (Lhoerion) [inline docs/articles/tables/weather.md:1]: Definitely not

File: docs/articles/tables/vehicle-bones.md [hunk 17]
```diff
@@ -0,0 +1,457 @@
+
```
Comment (Lhoerion) [inline docs/articles/tables/vehicle-bones.md:1]: Definitely not

Comment (Lhoerion) [inline docs/articles/tables/vehicle-bones.md:1]: Definitely not

File: docs/articles/tables/vehicle-colors.md [hunk 18]
```diff
@@ -0,0 +1,168 @@
+
```
Comment (Lhoerion) [inline docs/articles/tables/vehicle-colors.md:1]: Definitely not

File: docs/articles/tables/weapons.md [hunk 19]
```diff
@@ -0,0 +1,1337 @@
+# List of Weapons and Hashes
```
Comment (Lhoerion) [inline docs/articles/tables/weapons.md:1]: Definitely not

File: docs/articles/snippets/client/create-help-text.md [hunk 20]
```diff
@@ -1,10 +1,10 @@
-# Help Text
```
Comment (Lhoerion) [inline docs/articles/snippets/client/create-help-text.md:1]: Docs is not for code snippets

File: docs/articles/snippets/client/create-markers.md [hunk 21]
```diff
@@ -0,0 +1,76 @@
+# Create Markers
```
Comment (Lhoerion) [inline docs/articles/snippets/client/create-markers.md:1]: Docs is not for code snippets

File: docs/articles/snippets/client/create-notifications.md [hunk 22]
```diff
@@ -1,14 +1,17 @@
-# Notifications
```
Comment (Lhoerion) [inline docs/articles/snippets/client/create-notifications.md:1]: Docs is not for code snippets

File: docs/articles/snippets/client/create-textlabel.md [hunk 23]
```diff
@@ -1,4 +1,4 @@
-# Drawing Text 3D & 2D
```
Comment (Lhoerion) [inline docs/articles/snippets/client/create-textlabel.md:1]: Docs is not for code snippets

File: docs/articles/snippets/client/flickering-webview.md [hunk 24]
```diff
@@ -0,0 +1,16 @@
+# Flickering Webview
```
Comment (Lhoerion) [inline docs/articles/snippets/client/flickering-webview.md:1]: Docs is not for code snippets

File: docs/articles/snippets/client/index.md [hunk 25]
```diff
@@ -0,0 +1,3 @@
+# About Client Snippets
```
Comment (Lhoerion) [inline docs/articles/snippets/client/index.md:1]: Docs is not for code snippets

File: docs/articles/snippets/client/request-model.md [hunk 26]
```diff
@@ -0,0 +1,52 @@
+# Request Model
```
Comment (Lhoerion) [inline docs/articles/snippets/client/request-model.md:1]: Docs is not for code snippets

File: docs/articles/snippets/client/spotlight.md [hunk 27]
```diff
@@ -0,0 +1,42 @@
+# Create Spotlight
```
Comment (Lhoerion) [inline docs/articles/snippets/client/spotlight.md:1]: Docs is not for code snippets

File: docs/articles/snippets/client/use-local-storage.md [hunk 28]
```diff
@@ -1,4 +1,4 @@
-# Introduction to the LocalStorage
```
Comment (Lhoerion) [inline docs/articles/snippets/client/use-local-storage.md:1]: Docs is not for code snippets

File: docs/articles/snippets/server/forward-vector.md [hunk 29]
```diff
@@ -0,0 +1,31 @@
+# Forward Vector for Server Side
```
Comment (Lhoerion) [inline docs/articles/snippets/server/forward-vector.md:1]: Docs is not for code snippets

File: docs/articles/snippets/server/index.md [hunk 30]
```diff
@@ -0,0 +1,3 @@
+# About Server Snippets
```
Comment (Lhoerion) [inline docs/articles/snippets/server/index.md:1]: Docs is not for code snippets

File: docs/articles/snippets/server/password-hashing.md [hunk 31]
```diff
@@ -1,12 +1,12 @@
-# Encryption / Password Hashing
```
Comment (Lhoerion) [inline docs/articles/snippets/server/password-hashing.md:1]: Docs is not for code snippets

File: docs/articles/snippets/server/prototyping.md [hunk 32]
```diff
@@ -1,16 +1,18 @@
-# Prototyping
```
Comment (Lhoerion) [inline docs/articles/snippets/server/prototyping.md:1]: Docs is not for code snippets

File: docs/articles/snippets/utility/index.md [hunk 33]
```diff
@@ -0,0 +1,3 @@
+# About Utility Snippets
```
Comment (Lhoerion) [inline docs/articles/snippets/utility/index.md:1]: Docs is not for code snippets

File: docs/articles/snippets/utility/sha256.md [hunk 34]
```diff
@@ -0,0 +1,105 @@
+# Pure JS SHA256
```
Comment (Lhoerion) [inline docs/articles/snippets/utility/sha256.md:1]: Docs is not for code snippets

**Review by Lhoerion — CHANGES_REQUESTED** [review]
Basically articles sections `snippets` and `tables` are definitely not worth adding from my point of view

**Pull request closed without merging**